.config/umd.js

```diff
@@ -0,0 +1,18 @@
+import { addLicenseBanner } from './helpers/licenseBanner';
+
+const env = process.env.NODE_ENV;
+const envHotType = process.env.HOT_TYPE;
+const filename = 'react-handsontable.js';
+
+const umdConfig = {
+  output: {
+    format: env,
+    name: 'Handsontable.react',
+    indent: false,
+    file: './dist/' + envHotType + '/' + filename
+  }
+};
+
+addLicenseBanner(umdConfig);
+
+export { umdConfig };
```

Comment:
Could you add an empty line below?

Reply:
It has an empty last line, github just doesn't show it.